.travis.yml

```diff
@@ -56,6 +56,14 @@ jobs:
         UBUNTU: "20.04"
         cares_SOURCE: "BUNDLED"
         gRPC_SOURCE: "BUNDLED"
+    - name: "Go on s390x"
+      os: linux
+      arch: s390x
+      env:
+        ARCH: s390x
+        ARROW_CI_MODULES: "GO"
+        DOCKER_IMAGE_ID: debian-go
+        GO: "1.12"
```

Comment:
Could you remove this to use the default value in `.env`?

Reply:
Thanks for your feedback. I've removed the line.